src/components/ButtonDropdown/ButtonDropdown.tsx

```diff
@@ -189,18 +176,13 @@ export const ButtonDropdown = ({
     className,
   );
   return (
-    <div
-      className={componentClassName}
-      /* TODO: Figure out role that allows blur to entire menu
-      /* eslint-disable-next-line jsx-a11y/no-static-element-interactions */
-      onBlur={handleBlur}
-      ref={ref}
-      {...other}
-    >
+    <div className={componentClassName} ref={ref} {...other}>
       {dropdownMenuTriggerWithProps}
       <DropdownMenu
         className={styles['button-dropdown__dropdown-menu']}
-        handleOnEscDown={(e: React.KeyboardEvent) => handleKeyDown(e)}
+        handleOnClick={closePanel}
```

Comment:
Would have loved to have just used the existing event listeners in lines 106 but doing so prevents click events from being captured by the grandchildren `<DropdownMenuItem>` due to the native event handling occurring and dismounting the children before the React onClick could be called